src/components/shared/pageHeader/pageStats/index.js

```diff
@@ -0,0 +1,5 @@
+// Copyright (c) Microsoft. All rights reserved.
```

Comment:
These also be used on the dashboard page.

Reply:
logged a bug for this